built_value_generator/lib/src/serializer_source_class.dart

```diff
@@ -57,12 +65,21 @@ abstract class SerializerSourceClass
         element.fields.where((field) => field.name == 'serializer').toList();
     if (serializerFields.isEmpty) return '';
     final serializerField = serializerFields.single;
-    return serializerField.getter?.computeNode()?.toString() ?? '';
+    return parsedLibrary
+            .getElementDeclaration(serializerField.getter)
+            ?.node
+            ?.toSource() ??
+        '';
   }
 
   @memoized
-  BuiltList<String> get genericParameters => new BuiltList<String>(
-      element.typeParameters.map((e) => e.computeNode().toString()));
+  BuiltList<String> get genericParameters =>
+      new BuiltList<String>(element.typeParameters.map((e) {
+        final result = parsedLibrary.getElementDeclaration(e).node.toSource();
+        // Strip off any bounds.
```

Comment:
The original version you had gave different results, e.g. `T extends num` where previously I got `T`. So I just strip off any `extends` to get the same value.

Reply:
Ah, I guess I missed this place.
There was one more somewhere else.
You don't need to use code / string manipulations.

Something like
```dart
TypeParameter node = parsedLibrary.getElementDeclaration(e).node;
return node.name.name;
```

Or, even better, just
```
(e) => e.name;
```

I.e. if you need just name, get it from the element itself.

Reply:
That worked, thanks!